Eslint to project

Omotola28 opened the pull request:
Installed eslin to project 
Fixed eslint suggestions in App.js and serviceworker.js

Omotola28 committed:
Added extra information that might be useful for starting in readme file
- When setting up for the first time user should install yarn
- User needs to run yarn install to install exsisting packages

Omotola28 committed:
Eslint to project & Fixing code based on linting suggestions
- Installed eslint
- Configured eslint file in root directory called eslintrc
- Changed code in serviceWorker.js file based on suggestions from eslint
- Code completely linted

mazma1 commented on .eslintrc:
```diff
@@ -0,0 +1,15 @@
+    "semi": 0,
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
+  }
+}
```
This is missing a new line at the end so the file can breathe..

Omotola28 replied:
Alright I would add one and send this back up

mazma1 commented on package.json:
```diff
@@ -8,13 +8,15 @@
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "typescript": "^3.9.6"
```
Do we need TypeScript for anything?

Omotola28 replied:
No we do not must have mistakenly added it to the package.json file

mazma1 reviewed (commented):
@Omotola28 this is a good configuration to start with. I left a few comments; nothing much and we should be good to go

Omotola28 committed:
Made modifications based on comments made
- Added new line to .eslintrc file
- Removed typescript as a dependency from package.json file
